docs/release-notes/Percona-Everest-1.10.0-(2025-10-23).md

```diff
@@ -0,0 +1,145 @@
+    5. Click **Create**. Your Split-Horizon DNS policy will be created and appears in the configuration list.
+        
+## New features
+
+- [EVEREST-2239](https://perconadev.atlassian.net/browse/EVEREST-2239): Added Split-Horizon DNS support for Percona Server for MongoDB (PSMDB).
+
+- [EVEREST-2009](https://perconadev.atlassian.net/browse/EVEREST-2009): Percona Everest now supports Percona PostgreSQL Operator v2.7.0.
+
+- [EVEREST-2173](https://perconadev.atlassian.net/browse/EVEREST-2173): Percona Everest now supports Percona Operator for MongoDB 
+v1.21.0.
+
+- [EVEREST-2174](https://perconadev.atlassian.net/browse/EVEREST-2174): Percona Everest now supports Percona XtraDB Cluster Operator v1.18.0.
+
+
+## Improvements
+
+- [EVEREST-1853](https://perconadev.atlassian.net/browse/EVEREST-1853): Percona Everest now supports Percona Monitoring and Management (PMM) v3, giving you access to the latest monitoring, alerting, and performance insights. PMM v2 remains supported for backward compatibility.
+
+- [EVEREST-1889](https://perconadev.atlassian.net/browse/EVEREST-1889): The **Overview → Resources** section now displays the number of nodes per shard, providing a clearer understanding of resource allocation in sharded deployments at a glance.
+
+
+- [EVEREST-2236](https://perconadev.atlassian.net/browse/EVEREST-2236): Expose proxy vars in Helm values.
+
+- [EVEREST-2302](https://perconadev.atlassian.net/browse/EVEREST-2302): Menu informational sections are now displayed as plain text. They no longer behave like actions (clickable or hoverable), improving clarity and reducing confusion.
+
+- [EVEREST-2318](https://perconadev.atlassian.net/browse/EVEREST-2318): The dropdown option **Load Balancer** has been renamed to **LoadBalancer** to align with other service types, such as **ClusterIP** and **NodePort**.
+
+- [EVEREST-2323](https://perconadev.atlassian.net/browse/EVEREST-2323): The DB Importers UI now displays credentials in the same order as they appear in Kubernetes secrets and documentation, ensuring a consistent workflow across CLI, docs, and UI.
+
+
+## Bug fixes
+
+- [EVEREST-1865](https://perconadev.atlassian.net/browse/EVEREST-1865): **Point-in-Time Recovery (PITR) details** are now displayed correctly when editing PostgreSQL database clusters.
+
+- [EVEREST-1947](https://perconadev.atlassian.net/browse/EVEREST-1947): Fixed an issue where the **Copy URL to Clipboard** button in the database details view was not working. Users can now successfully copy the connection URL.
+
+
+- [EVEREST-2023](https://perconadev.atlassian.net/browse/EVEREST-2023): The `everest operator` showed inconsistent behavior in handling resource creation and updates. This issue has now been resolved.
+
+
+- [EVEREST-2109](https://perconadev.atlassian.net/browse/EVEREST-2109): Point-in-Time Recovery (PITR) now correctly activates for PostgreSQL clusters after creating on-demand or scheduled backups, addressing the previous issue where it remained disabled.
```

Comment:
We should rephrase this because this is only a presentation issue, the PITR was enabled and able to be used but the UI showed differently.